Gruntfile.js

```diff
@@ -293,14 +324,25 @@ module.exports = function(grunt) {
       }
     },
     usebanner: {
```

Comment:
I saw that but wanted the banner in the non-minified version too, including version and copyright. Sounds like we could use browserify to do var replacement here too instead.